src/tests/ftest/cart/test_group_np_common.h

```diff
@@ -54,13 +73,15 @@ struct test_t test_g = { .t_hold_time = 0,
 			 .t_srv_ctx_num = 1,
 			 .t_roomno = 1082 };
 
-#define CRT_ISEQ_TEST_PING_CHECK /* input fields */		 \
+/* input fields */
+#define CRT_ISEQ_TEST_PING_CHECK				 \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
Ignoring: `checkpatch.pl` can't seem to analyze complex macros properly.